python/eos/data/native.py

```diff
@@ -340,7 +340,7 @@ def create(path, parameters, proposal, sigma_test_stat=None, samples=None, weigh
 
         # The test statistics defaults to two empty lists
         description['test statistics'] = { "sigma": [], "densities": [] }
-        if sigma_test_stat and samples and weights:
+        if not sigma_test_stat is None and not samples is None and not weights is None:
             sigma_test_stat = _np.array(sigma_test_stat)
             samplesPDF = [x for x in map(lambda x: -2.0 * _np.log(PMCSampler._evaluate_mixture_pdf(proposal, x)), samples)]
```

Comment:
Unrelated, but this looks nasty...

```python
samplesPDF = list(map(lambda x: -2.0 * _np.log(PMCSampler._evaluate_mixture_pdf(proposal, x)), samples))
```
does the trick, doesn't it?

Reply:
probably :)